robot-command/src/main/java/org/obolibrary/robot/RemoveCommand.java

```diff
@@ -198,12 +202,40 @@ public CommandState execute(CommandState state, String[] args) throws Exception
               copy, baseNamespaces, complementObjects, anonymous, internal, external));
     }
 
+    // remove specified axiom annotations
+    dropAxiomAnnotations(line, ioHelper, ontology);
+
     // Save the changed ontology and return the state
     CommandLineHelper.maybeSaveOutput(line, ontology);
     state.setOntology(ontology);
     return state;
   }
 
+  /**
+   * Given a command line and an IOHelper, drops axiom annotations from the given ontology which
+   * uses annotation properties specified by the drop_axiom_annotation argument.
+   *
+   * @param line CommandLine to get options from
+   * @param ioHelper IOHelper to get IRIs
+   * @param ontology OWLOntology to drop axiom annotations from
+   */
+  private void dropAxiomAnnotations(CommandLine line, IOHelper ioHelper, OWLOntology ontology) {
```

Comment:
Why isnt this method in OntologyUtils or some such? This should not be duplicated across both commands?

Reply:
generic methods moved to `OntologyHelper.java` and methods only used by the FilterCommand and RemoveCommand are moved to `RelatedObjectsHelper.java`